atsign-foundation/noports
# feat: (c sshnpd) daemon npt support

**- What I did**

- @realvarx had an initial mvp of npt request
- Took that mvp, and merged all of the changes in trunk & latest at_c
- diffed handle_ssh_request.c and handle_npt_request.c making the implementations identical wherever possible, differences are only where the actual implementation **should** differ

**- How I did it**

**- How to verify it**

see also: https://github.com/atsign-foundation/at_c/pull/402

Had a demo, leaked my password, can upload a new demo tomorrow...

e2e test coverage now covers npt to port 22 & c daemon.

**- Description for the changelog**
feat: (c sshnpd) daemon npt support
